client/gutenberg/extensions/shortlinks/panel.js

```diff
@@ -0,0 +1,39 @@
+/** @format */
```

Comment:
We can drop Prettier pragmas since https://github.com/Automattic/wp-calypso/pull/29403

Reply:
👍 Good point, removed in https://github.com/Automattic/wp-calypso/pull/29475/commits/1e3979b42c6a321f7cbe114ff47347f9c203f690